DTS-STN/Alpha-Site
# QA-unsubscribe

# Description

[#377 Unsubscribe Pages EN/FR](https://trello.com/c/97qjL7SL/377-377-unsubscribe-pages-en-fr)

Updated the unsubscribe translations. French and English.

## Acceptance Criteria

The text must match the Figma design.

## Test Instructions

1. Start by subscribing a email
2. once subscribed, follow the unsubscribe flow in both french and english.
3. Make sure all text matches the Figma design as well as the sizing of the text.
4. All text must also fit within the page.

## Help Requested

- I need help modifying the content sent back to the user in the unsubscribe email.
**What we have:**
![image](https://user-images.githubusercontent.com/77116011/126194410-ecbad986-27a9-4e2c-9e7a-2c98f8e274f9.png)

**What we need:**
![image](https://user-images.githubusercontent.com/77116011/126194508-d2510ba8-9e67-41ee-80f2-a7a21fe9fbd8.png)

## Checklist
